project/source/Scripts/Objects/LabObject.gd

```diff
@@ -155,7 +155,7 @@ func GetIntersectingLabObjects():
 	#For each collider that we have, see if it colllides with anything.
 	#This way, we could have LabObjects with multiple colliders on them (for example, multiple rectangles making up a more complex shape)
 	for child in get_children():
-		if child is CollisionShape2D or child is CollisionPolygon2D:
```

Comment:
Nice job noticing this!

I actually thought I had added support for CollisionPolygon2Ds a _while_ ago (like, in the fall semester).

I'd still like to add support for that, since I think it should just involve constructing a new `shape` for the query params from the vertices of a CollisionPolygon2D, but I'm not 100% sure anymore. If I do have time to add that support, it should be a separate branch from this one though, so this change is definitely good.